README.md

```diff
@@ -1,21 +1,416 @@
-FHIR Proxy’s SMART scoping has many benefits for FHIR Server – the major one being that it brings FHIR Server into compliance with HL7’s SMART App Launch Framework. With SMART scoping, the permissions delegated to a SMART app can be federated to the app’s end-users (defined in HL7’s SMART App Launch Implementation Guide as “patients” or “users”) without having to assign each end-user a role in the AAD tenant. Furthermore, once the SMART scopes have been defined and set up as API endpoints on FHIR Proxy, it is straightforward to create scope definitions for different SMART apps. This can greatly simplify the scoping of client permissions down to specific Resources (FHIR) – requiring no special configuration on the server side when grouping permissions for a custom “scope” (for example, for a client’s boutique SMART app). Moreover, the OAuth 2.0 authentication scheme gives a means of issuing client secrets to trusted SMART apps, thus hardening FHIR data security.
+Check back often as more processing modules will be added. </br>
+ 
+See [Proxy Configuration](##configuration) section below for full descriptions and configuration instructions.
+## Reverse Proxy
+All FHIR Server responses are re-written to include the proxy address as the appropriate endpoint so the FHIR Server URL is never directly exposed.
 
-## Triggerable Event Publishing
-FHIR Proxy also extends FHIR Server functionality with triggers that can be set to listen for Create, Update, and Delete (CUD) actions performed on FHIR Data. Administrators can set these triggers in the FHIR Proxy to automatically publish to Azure Event Hub when a CUD action occurs in the FHIR Server. This functionality is important for enabling near-real-time Clinical Decision Support (CDS)         via HL7 CDS Hooks across Care Teams. The recipient(s) could be anyone among the FHIR Participants permitted to subscribe to the alerts, including the Patient.
+## Architecture Overview
+![Fhirproxy Arch](https://github.com/microsof/fhir-proxy/docs/images/architecture/fhirproxy_arch.png)
+
+
+## Deploying your own FHIR Proxy
+
+Please note you should deploy this proxy into a tenant that you control Application Registrations, Enterprise Applications, Permissions and Role Definitions Assignments
```

Comment:
Maybe rephrase to say "..that you have appropriate permissions to create and manage Application Registrations, Enterprise Applications, Permissions and Role Definition Assignments".